autoload/vital/_ghpr_blame/Web/HTTP.vim

```diff
@@ -200,7 +200,7 @@ function! s:_build_response(header, content) abort
 
   if !empty(a:header)
     let status_line = get(a:header, 0)
-    let matched = matchlist(status_line, '^HTTP/1\.\d\s\+\(\d\+\)\s\+\(.*\)')
+    let matched = matchlist(status_line, '^HTTP/2\s\+\(\d\+\)\s\+\(.*\)')
```

Comment:
Please support both 1 and 2
```suggestion
    let matched = matchlist(status_line, '^HTTP/\%(2\|1\.\d\)\s\+\(\d\+\)\s\+\(.*\)')
```

I don't know all APIs use HTTP2. For example some documentation shows HTTP 1.1 response:

> HTTP/1.1 200 OK

https://docs.github.com/en/rest/overview/resources-in-the-rest-api#schema

Reply:
I applied your suggested changes.